.github/workflows/truffleruby.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Setup system Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 3.2
```

Comment:
This should still be 3.2 or at least a release (`ruby-version: ruby`), this ruby will notably be used to run `jt`, which might not work on ruby-head.